index.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const Redis = require('ioredis'),
+  redis = new Redis(process.env.CLAY_BUS_HOST),
+  os = require('os'),
+  HOSTNAME = os.hostname();
+var log = require('./log').setup({ file: __filename });
+
+function connect() {
+  log('info', `Connecting to Redis Event Bus at ${process.env.CLAY_BUS_HOST}`);
+  module.exports.client = new Redis(process.env.CLAY_BUS_HOST);
+  return { publish };
+}
+
+function publish(topic, msg) {
+  if (module.exports.client) {
+    redis.publish(topic, JSON.stringify({
+      hostname: HOSTNAME,
```

Comment:
For the purposes of this release it's not required, but it would be nice to start generating message ids here.

Reply:
Ohhhhh can you make an issue real fast?

Reply:
Sure, https://github.com/clay/amphora-event-bus-redis/issues/2